apps/daimo-mobile/src/logic/onboarding.ts

```diff
@@ -0,0 +1,46 @@
+import { SlotType, getSlotType } from "@daimo/common";
+import { useCallback, useContext, useMemo } from "react";
+
+import { DispatcherContext } from "../action/dispatch";
+import { useNav } from "../common/nav";
+import { Account } from "../model/account";
+
+export function useOnboardingChecklist(account: Account) {
+  const nav = useNav();
+  const dispatcher = useContext(DispatcherContext);
+
+  const hasBackup = account.accountKeys.some(
+    (key) => getSlotType(key.slot) === SlotType.PasskeyBackup
+  );
+
+  const farcasterConnected = !!account.linkedAccounts.find(
+    (a) => a.type === "farcaster"
+  );
+
+  const allComplete = useMemo(() => {
+    return hasBackup && farcasterConnected;
+  }, [hasBackup, farcasterConnected]);
+
+  const handleSecureAccount = useCallback(() => {
+    nav.navigate("Settings");
```

Comment:
hm should we just the user directly to the passkey backup page, not just settings?

Reply:
I think that's a good idea. Adding in the next commit